spacy/scorer.py

```diff
@@ -1143,3 +1143,144 @@ def _auc(x, y):
         # regular numpy.ndarray instances.
         area = area.dtype.type(area)
     return area
+
+
```

Comment:
Can you add a comment header here to separate this code from the preceding scikit-learn code (just similar to what's there for ROC AUC)?

Reply:
Added in the most recent commit.